lib/ash_json_api/json_schema/json_schema.ex

```diff
@@ -683,6 +684,17 @@ defmodule AshJsonApi.JsonSchema do
     end)
   end
 
+  defp required_relationship_attributes(resource, accept) do
+    resource
+    |> AshJsonApi.Resource.fields()
+    |> Enum.filter(&(is_nil(accept) || &1 in accept))
+    |> Enum.filter(fn field ->
+      rel = Ash.Resource.relationship(resource, field)
+      !is_nil(rel) && rel.required?
```

Comment:
Some relationships don't have have the `required?` field, so we'll want to use `!is_nil(rel) &&  Map.get(rel, :required?)`

Reply:
Ok.